CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Reduce memory usage when committing write transactions.
+* Improve performance of compacting encrypted Realm files.
+* Throw an exception when opening a Realm file with an invalid top ref rather
+  than crashing with an assertion failure.
 
 ### Fixed
 * Fix a memory leak whenever Realm makes a HTTP(s) request to the Realm Object
   Server (since 3.8.0, Issue [#6058](https://github.com/realm/realm-cocoa/issues/6058)).
+* Fix an assertion failure when creating an object after creating an object
+  with a null int primary key in the same write transaction.
+  ([#3227](https://github.com/realm/realm-core/pull/3227)).
+* Properly clean up sync sessions when the last Realm object using the session
```

Comment:
How could a user experience this? 
Issue or PR #?

Reply:
No idea how a user would hit it or even notice. I discovered it was broken while writing tests for the audit functionality, so there's no issue and the PR has nothing to do with it.

Reply:
Then it should be in the ### internal section